src/xcode/ENA/ENA/Source/Scenes/Checkin/QRCodeScanner/CheckinQRCodeScannerViewController.swift

```diff
@@ -96,23 +197,54 @@ class CheckinQRCodeScannerViewController: UIViewController {
 
 		viewModel.deactivateScanning()
 
-		let alert = UIAlertController(title: "QR-Code nicht gültig", message: nil, preferredStyle: .alert)
+		let alert = UIAlertController(
+			title: AppStrings.Checkin.QRScanner.Error.title,
+			message: AppStrings.Checkin.QRScanner.Error.description,
+			preferredStyle: .alert
+		)
 		alert.addAction(
 			UIAlertAction(
-				title: "abbrechen",
+				title: AppStrings.Common.alertActionCancel,
 				style: .cancel,
 				handler: { [weak self] _ in
 					self?.dismiss()
 				}
 			)
 		)
-		alert.addAction(UIAlertAction(title: "OK", style: .default, handler: { [weak self] _ in
+		alert.addAction(UIAlertAction(title: AppStrings.Common.alertActionOk, style: .default, handler: { [weak self] _ in
 			self?.viewModel.activateScanning()
 		}))
 
 		DispatchQueue.main.async { [weak self] in
 			self?.present(alert, animated: true)
 		}
 	}
+	private func setNeedsPreviewMaskUpdate() {
```

Comment:
This func is not needed anymore. Please call updatePreviewMaskIfNeeded() directly. And also I would suggest to rename "updatePreviewMaskIfNeeded" to "updatePreviewMask".

Reply:
done